index.html

```diff
@@ -0,0 +1,688 @@
+            </svg>
+        </a>
+</div>
+<p class="endnote">A version of this article appears in print on March 18, 2014, Section A, Page 1 of the New York edition with the headline: Space Ripples Reveal Big Bang’s Smoking Gun. <a class="navgt" href="#">Order Reprints</a>| <a class="navgt" href="#">Today’s Paper</a> | <a class="navgt" href="#">Subscribe</a></p>
+    </article></div>
+        <div class="right">
+            <aside>
+            <div class="editors-pick">
+                    <ul class="e-picklist">
+                        <li><h3 class="picklist">Editor's Pick</h3></li>
+                <li><a href="#">
+                        <img class="illustrations" src="images/17money-alt-square640.jpg" alt="">
+                        <p>Where Are All The Wild Things,
+                            Daddy?</p>
+                    </a></li>
+                <li><a href="#">
+                        <img class="illustrations" src="images/20MODERN-FATHERSDAY-square640.jpg" alt="">
+                        <p>From the Heart to Higher Education: The 2021 College <br> Essays on
+                            Money</p>
+                    </a></li>
+    
+                <li><a href="#">
+                        <img class="illustrations" src="images/18surfacing-pet-tube-thmpsn1-square640.jpg" alt="">
+                        <p>The Internet’s Exotic Pets, and the People Who Love Them</p>
+                    </a></li>
+                </ul> </div>
+        </aside>
+    </div>
+    </div>
+
+    <h3 class="more">More in Space and Astronomy</h3>
+    <div class="headlines">
+        <div class="headbox">
+            <figure>
+            <img class="headline-img" src="images/Biden.jpg" alt="biden">
+        <figcaption><b>Biden Nominates Former Florida Senator To Lead NASA</b></figcaption>
+    </figure>
+        </div>
+        <div class="headbox">
+            <figure>
+            <img class="headline-img" src="images/mars.jpg" alt="mars">
+        <figcaption><b>The Water On Mars Vanished. This Might Be Where It Went</b></figcaption>
+        </figure></div>
+
+        <div class="popular">
+            <h3>Most Popular</h3>
+            <p><b>If Everything Smells Bad, You’re Not <br/> Alone</b></p>
+            <p><b>California’s First Latino Senator <br/> Wants Immigration Overhaul Now</b></p>
+            <p><b>$325,000 Settlement for Teacher <br/> Over Trump References Removed <br/> From Yearbook</b></p>
+            <p><b>The Private Lives of Strippers</b></p>
+            <p><b>I’m an Athlete, How Can I Find <br/> Clothes That Fit?</b></p>
+            <p><b>How the World’s Oldest Wooden <br/> Sculpture Is Reshaping Prehistory</b></p>
+            <p><b>A Documentary Explores <br/> Dissociative Identity Disorder</b></p>
+            <p><b>Tom Reed, Apologizing Over <br/> Groping Allegation, Says He Won’t <br/> Run in 2022</b></p>
+        <p><b>The Summers We Won’t See Again</b></p>
```

Comment:
- [ ] Please have a look at the original design of same template and arrange your **More in space astronomy** section properly.
- [ ] There's an overflow in your page, probably from width assignment or wrong position class, please fix this.

### TIP: You can ask for help from one of your course mates and have this resolved.